src/WireMock.Net/Util/BodyParser.cs

```diff
@@ -58,6 +59,7 @@ internal static class BodyParser
             else if (contentTypeHeaderValue != null && contentTypeHeaderValue.StartsWith("application/json", StringComparison.OrdinalIgnoreCase))
             {
                 var stringData = await ReadStringAsync(stream);
+                data.BodyAsStringOriginal = stringData.Item1;
```

Comment:
And then put Item1 into Body instead and remove BodyAsStringOriginal?

Reply:
Fixed